### DISCUSSION TITLE
Depth decoy shell is useless because it sinks too fast.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14493] [Ideas and suggestions]
The reasons are:
1. Depth decoy shell launched from the tube sinks too fast. The decoy almost instantly leaves the monsters' line of sight so the monsters that already see the submarine do not have time to switch their attention to the decoy and continue to attack the submarine.
2. when colliding with the seabed, the decoy is destroyed.
I suggest:

a) Lets make Depth decoy shell  sinking speed very slow or adjustable.
b) Depth decoy shell  should be destroyed upon hitting the seabed only if it contains an explosive charge inside. If there is no explosive charge in the decoy, then it can be picked up and reused.
c) Lets activate decoy internal beacon even if decoy was just dropped out of sub hatch with hand. This will allow to use the decoy even on boats that do not have a depth charge launcher .

**Comment by SomeRandomNoobKekeke** [comment]
> 2. when colliding with the seabed, the decoy is destroyed.

It's not, but it gets destroyed in one bite by almost any creature

> 5. Lets activate decoy internal beacon even if decoy was just dropped out of sub hatch with hand. This will allow to use the decoy even on boats that do not have a depth charge launcher.

That's just regular sonar beacon

> Lets make Depth decoy shell sinking speed very slow

Then it's not gonna divert abyss creatures, because they are very fast

##  Just mod it (tm)
[There are no good, up to date mods for better decoys](https://steamcommunity.com/workshop/browse/?appid=602960&searchtext=decoy&childpublishedfileid=0&browsesort=textsearch&section=), so go make one

Make harder / lighter versions, decoys that activates after delay

I personally request cluster decoys

**Reply by pavel4win** [reply]
> > 2. when colliding with the seabed, the decoy is destroyed.
> 
> It's not, but it gets destroyed in one bite by almost any creature

I have just dropped the decoy to the seabed to check. 
The decoy is destroyed when it hits the bottom. You can't pick it up and reuse.

Will decoy detonate (if it have explosive charge inside) if the monster bites it?
I will check it soon.
> 
> > 5. Lets activate decoy internal beacon even if decoy was just dropped out of sub hatch with hand. This will allow to use the decoy even on boats that do not have a depth charge launcher.
> 
> That's just regular sonar beacon

It is NOT "just regular sonar beacon" because decoy can contain explosive. But sonar beacon can not.

If the decoy or Depth charge is thrown out of a hatch manually, it will not detonate or even collide with monsters. 
In "manual thrown" case, the decoy falls slowly, lies on the bottom, and can be taken back. But it does not distract anyone, since the built-in beacon is not activated.

> > Lets make Depth decoy shell sinking speed very slow
> Then it's not gonna divert abyss creatures, because they are very fast

Adjustable sinking speed will be the best solution.
The speed must be tuned up in the bomb/decoy itself or in the launcher or both.

> ## Just mod it (tm)
> [There are no good, up to date mods for better decoys]

I'll think about it, thanks.

**Reply by SomeRandomNoobKekeke** [reply]
> Will decoy detonate (if it have explosive charge inside) if the monster bites it?

It should, what else is that slot for?

> The speed must be tuned up in the bomb/decoy itself or in the launcher or both.

Imagine captain running to gunnery compartment to adjust decoy sinking speed manually while being chased by charybdis

**Reply by pavel4win** [reply]
This is a game moment.
The bomb should be set up not by the captain, but by the Security.
The bomb can be set up in advance.
You can set up not the bomb but the launcher, if you know what kind of beast you will have to fight.

What I mean is that the setting of the sunk speed of the bomb should not happen at the moment of launch. It should be a property of the bomb that can be set once and it should be saved between rounds for each bomb.

**Comment by ubertpendragon** [comment]
It's too fast tbh

**Comment by Ek-Videogames** [comment]
Yeah decoys are weird.

A possible solution to the issue of a 'slower' decoy not drawing monsters away is **make the actual lure activate on a delay** so it's at least somewhat away from the sub by the time it turns on.

Or if buoyancy is a serializable value (or make it one), you can have it start dense and then decelerate to a slightly-closer-to-neutral buoyancy after this time. 

( Not sure if this was changed at all, but at one point didn't depth charge launchers give the depth charge a bit of launch velocity in order to get it away from the sub faster? )

**Reply by SomeRandomNoobKekeke** [reply]
> Or if buoyancy is a serializable value

I think it's density 
![Screenshot_3](https://github.com/user-attachments/assets/9c708c6f-79fd-4cc2-b32e-2118598093a8)

> at one point didn't depth charge launchers give the depth charge a bit of launch velocity

Ye DepthCharge.Projectile.launchimpulse, clearly a thing since i have sub that launches them upwards to increase it's lifetime :BaroDev:

**Reply by ubertpendragon** [reply]
yeah density is the issue it's what makes it sink after the initial impulse

**Comment by BlackMoons** [comment]
Im actually working on a depth charge mod with slower sink rates tuned to let creatures catch up (And blow up in their face, taking out half the pack if loaded with C4)

Also, floating decoy depth charges so you can have launchers pointed up too. Maybe i'll see about releasing it today.

PS: I found that 10.5 (for sinking) and 9.5 (for floating) provide very usable decoy charges that eventually let most enemies catch up to.

**Reply by pavel4win** [reply]
After the depth charge mod release publish the link here?

**Reply by BlackMoons** [reply]
Here is the mod I made for depth charges that shows off slower decent rates 10.4 was chosen to allow mudraptors to catch up easily enough. Slightly more might be desirable for a distraction decoy. https://steamcommunity.com/sharedfiles/filedetails/?id=3319277391

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/issues/11788. But yeah, I agree, their speeds make them almost useless for their intended purpose, and we should address that.
